<launch>
    <group ns="NavigationSwitches">
        <param name="defaultSetting" type="bool" value="true"/>
        <param name="bIsSimulation" type="bool" value="false"/>
        <param name="bIsIntegralEnable" type="bool" value="true"/>
        <param name="bIsYawControlEnable" type="bool" value="false"/>
        <param name="bIsYawControlAlwaysAlign" type="bool" value="true"/>
        <param name="bIsYawControlEnableSearch" type="bool" value="false"/>
        <param name="bIsMPCEnable" type="bool" value="true"/>
        <param name="bIsLQREnable" type="bool" value="false"/>
        <param name="bIsKeepLanding" type="bool" value="true"/>
    </group>
    <group ns="KalmanFilterParameters">
        <param name="defaultSetting" type="bool" value="true"/>
        <param name="sigma_ax" type="double" value="0.1"/>
        <param name="sigma_ay" type="double" value="0.1"/>
        <param name="sigma_GPSpx" type="double" value="0.3"/>
        <param name="sigma_GPSpy" type="double" value="0.3"/>
        <param name="sigma_GPSvx" type="double" value="0.18"/>
        <param name="sigma_GPSvy" type="double" value="0.18"/>
        <param name="sigma_Apriltagpx" type="double" value="0.1"/>
        <param name="sigma_Apriltagpy" type="double" value="0.1"/>
        <param name="nPred" type="int" value="2"/>
    </group>
    <node name="navigation_node" pkg="navigation" type="navigation_node" output="screen" cwd="node">
    </node>
</launch>


<!-- the "default setting" element is to easily compare with the default setting. If it's true, then all the later setting is invalid -->
